src/temporary_home/ReactiveSocket.h

```diff
@@ -80,6 +82,11 @@ class ReactiveSocket {
       Payload payload,
       yarpl::Reference<yarpl::flowable::Subscriber<Payload>> responseSink);
 
+  bool requestStream(
+      Payload payload,
+      yarpl::Reference<yarpl::flowable::Subscriber<Payload>> responseSink,
+      std::string streamName);
```

Comment:
I dislike the term `streamName`, I liked the name `streamToken` but maybe there's something more descriptive.